roles/bwc_repos/tasks/bwc_repos_yum.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: Assert that master_token is specified
+  fail: msg="License key must be supplied for BWC enterprise installation."
```

Comment:
`fail` check is duplicated in both `bwc_repos_yum.yml` and `bwc_repos_apt.yml` files.
Sounds cool if we can juse move it to parent `main.yml` file.